pmd-dist/src/test/java/net/sourceforge/pmd/it/BinaryDistributionIT.java

```diff
@@ -18,16 +18,27 @@
 import org.junit.Test;
 
 import net.sourceforge.pmd.PMDVersion;
-import net.sourceforge.pmd.cli.internal.CliMessages;
 
 public class BinaryDistributionIT extends AbstractBinaryDistributionTest {
 
     private static final String SUPPORTED_LANGUAGES_CPD;
     private static final String SUPPORTED_LANGUAGES_PMD;
 
     static {
-        SUPPORTED_LANGUAGES_CPD = "Supported languages: [apex, cpp, cs, dart, ecmascript, fortran, gherkin, go, groovy, html, java, jsp, kotlin, lua, matlab, modelica, objectivec, perl, php, plsql, python, ruby, scala, swift, vf, xml]";
-        SUPPORTED_LANGUAGES_PMD = "apex, ecmascript, html, java, jsp, kotlin, modelica, plsql, pom, scala, swift, vf, vm, wsdl, xml, xsl";
+        SUPPORTED_LANGUAGES_CPD = "Valid values: apex, cpp, cs, dart, ecmascript," + System.lineSeparator()
+                + "                               fortran, gherkin, go, groovy, html, java, jsp," + System.lineSeparator()
+                + "                               kotlin, lua, matlab, modelica, objectivec, perl," + System.lineSeparator()
+                + "                               php, plsql, python, ruby, scala, swift, vf, xml";
+        SUPPORTED_LANGUAGES_PMD = "Valid values: apex-54, ecmascript-ES6, html-," + System.lineSeparator()
```

Comment:
this looks a bit weird: "html-". Maybe we can improve this in a later PR (enforcing a real version for each language?)

Reply:
yes, this gets fixed as soon as we do #4120, that's why I left it this way for the time being. Little value in adding code to "fix" this on pmd-cli and then have to remove it again.